types/compilation/compilation.interfaces.ts

```diff
@@ -34,6 +34,7 @@ import {LLVMOptPipelineOutput} from './llvm-opt-pipeline-output.interfaces.js';
 export type CompilationResult = {
     code: number;
     timedOut: boolean;
+    okToCache?: boolean;
```

Comment:
I think this is right, good catch

Reply:
Thanks for checking, that was the most worrying bit :)